modules/apps/web-experience/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_sharing.jsp

```diff
@@ -79,6 +73,12 @@ String widgetURL = PortalUtil.getWidgetURL(portlet, themeDisplay);
 							<textarea class="field form-control lfr-textarea" id="<portlet:namespace />widgetScript" onClick="this.select();" readonly="true"><%= HtmlUtil.escape(textAreaContent) %></textarea>
 						</aui:field-wrapper>
 
+						<c:if test='<%= Validator.isNotNull(xFrameOptionsHeader) && (xFrameOptionsHeader.indexOf("ALLOW-FROM") == -1) %>'>
```

Comment:
Hey @jonmak08,  I think this might be incorrect.  We want to show the message if the header is not set, and also if it's set to something that does not contain `ALLOW-FROM`.  I think the way this is working is, if the property is not set the user would not see the message.